[IMP] inventory: thd batch updates

StraubCreative opened the pull request:
Overview
---------
This is a batch cherry pick of PR’s #2002, #1984, #1900, #1876, #1870, #1854, and #1824
I took the changes originally proposed in these PR’s and stacked them here on a fresh 15.0 base since they were all merge requests into 15.0.

Formatting updates
--------------------
On top of the original changes proposed by @thomasdeleval, I went ahead and updated technicals for each affected RST file + associated images. Those changes are as follows in order to bring each doc up to standards:

- fixed 100th character line breaks
- added alt text to images with proper grammar
- fixed indentations on RST tags
- renamed images attached to respective RST files (when appropriate) to match new convention (`usage_01.png` —> `uom-handling-vs-purchase.png`), and updated RST links accordingly.
- fixed some broken links + misnamed folders

Follow up items for review
--------------------------
I did not change any of the content other than what what the PRs originally suggested, however I think there are a number of improvements to be made around wording, grammar, and specificity of language. 

Generally speaking, please consider the following suggestions in content review stage:

- add missing `:guilabel:`'s 😈 
- wording/grammar: I caught a number of instances where the technical details were too wordy, vague, or the instruction got lost / wasn’t clear. Some words are strange like “hit” instead of “click” or “choose”.
- headings: especially the h1 so they follow convention [re:guidelines:headings](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings)
- Consider rephrasing all hypothetical narratives (e.g. “imagine if…”). Would stick to third-person instructional w/ time order language, if possible. “What if” scenarios are more helpful when visually illustrated with video or long-form blog articles, however for software documentation, this will interrupt ability to skim and absorb instructions which are supposed to be clear and easy to find.
- Similarly, would also reword/remove all first-person and second-person familiar statements and stick with third-person instructional.
- images: check all for HD resolution, at or around 768 pixel breakpoint in width, and are compressed. Are there any images that need to be reshot?
- images: are they actually showcasing the feature being written about? Some aren’t clear.
- remove unnecessary images, or surround them with new blocks of descriptive copy so the writing is leading the document instead of the visuals [re: guidlines:images](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/2556).

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst:
```diff
@@ -0,0 +1,59 @@
+You can see the different wave transfers through the menu :menuselection:`Inventory --> Operations
+--> Wave transfers`.
+
+.. image:: wave_transfers/waves-list.png
```
I would delete this image and its tags, it doesn't seem necessary.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst:
```diff
@@ -0,0 +1,59 @@
+   :align: center
+   :alt: confirm wave transfer
+
+You can see the different wave transfers through the menu :menuselection:`Inventory --> Operations
```
Delete the sentence on Lines 50-51 `You can see...Wave transfers'.` and replace with `To view all wave transfers and their statuses, go to :menuselection:'Inventory --> Operations --> Wave Transfers'. Wave transfers can also be viewed in the :guilabel:'Barcode' app by going to :menuselection:'Barcode --> Batch Transfers'.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst:
```diff
@@ -0,0 +1,59 @@
+   :align: center
+   :alt: View of a wave transfer list.
+
+In the barcode app, wave transfers are in the same view as batch transfers.
```
Delete this sentence.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst:
```diff
@@ -0,0 +1,59 @@
+
+In the barcode app, wave transfers are in the same view as batch transfers.
+
+Wave transfers are processed exactly like Batch transfers.
```
Delete this sentence.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/misc/wave_transfers.rst:
```diff
@@ -0,0 +1,59 @@
+Process Wave Transfers
+=======================
+
+While a Batch Transfer is a group of several pickings, a **Wave Transfer** only contains some parts
```
- Uncapitalize `Batch Transfer`
- Uncapitalize `Wave Transfer`

mivu-odoo reviewed (changes requested):
Finished reviewing content in `wave_transfers.rst`, will need to come back another time to review content in the other changed RST files.

For `wave_transfers.rst`, I fixed the wording to remove use of familiar second-person "you", added transition words and phrases to improve readability, fixed spelling and grammar issues, took new screenshots because the current images are wider than tablet breakpoint size, and left comments for which images can be removed.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+   :alt: A schedule of upcoming Inventory Adjustments and when they'll be applied in accounting.
+
+.. important::
+   In the barcode app, you will only see counts to make assigned to you, scheduled for today or
```
- Delete the one sentence on Lines 73-74 `In the barcode app...today or earlier`
- Replace with `In the Barcode app, users can only view counts assigned to them that are scheduled for today or earlier.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+   In the barcode app, you will only see counts to make assigned to you, scheduled for today or
+   earlier.
+
+If you want to plan big counts, the easiest way is to select some lines, then hit **Request A
```
- Delete the three sentences on lines 76-78 `If you want to...with the on hand quantity.`
- Replace with: 

> `To plan big counts, select the desired product lines on the :guilabel:'Inventory Adjustments' page. Then, click :guilabel:'Request a Count'. Fill in the following information:`
> `- :guilabel:'Inventory Date': the planned date of the count.`
> `- :guilabel:'User': the user responsible for the count.`
> `- :guilabel:'Accounting Date': the date at which the inventory adjustment will be accounted.`
> `- :guilabel:'Count': to leave the :guilabel:'On Hand Quantity' of each product line blank, select :guilabel:'Leave Empty'. To prefill the :guilabel:'On Hand Quantity' of each product line with the current value recorded in the database, select :guilabel:'Set Current Value'.`
> `Finally, click :guilabel:'Confirm' to request the count.`

StraubCreative replied:
Made small change

Instead of: 
```
Then, click :guilabel:`Request a Count`. Fill in the following information:
``` 

I did: 
```
Then, click :guilabel:`Request a Count` and fill in the following information:
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+   :align: center
+   :alt: Request A Count wizard allows the planning of large inventory counts.
+
+.. tip::
```
This tip on Lines 84-85 `The report offers a lot of filters that you can use to make your selection.` can be deleted. It is unnecessary.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+Count**. There you can select who will count and when. You can also prefill the quantity with the
+on hand quantity.
+
+.. image:: count_products/request-count.png
```
This image can be deleted. It is not necessary.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+.. tip::
+   The report offers a lot of filters that you can use to make your selection.
+
+By default, after an adjustment is applied, the scheduled date for the next count is the 31st of
```
Add `inventory` between `an` and `adjustment`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+   The report offers a lot of filters that you can use to make your selection.
+
+By default, after an adjustment is applied, the scheduled date for the next count is the 31st of
+December of the current year. This is a setting that can be modified in :menuselection:`Inventory
```
- Delete the one sentence on Lines 88-89 `This is a setting...Annual Inventory Day and Month.`
- Replace with: `To modify the default scheduled date, go to :menuselection:'Inventory --> Configuration --> Settings --> Operations' and change the date in the :guilabel:'Annual Inventory Day and Month' setting.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:
```diff
@@ -0,0 +1,93 @@
+December of the current year. This is a setting that can be modified in :menuselection:`Inventory
+--> Configuration --> Settings` under the setting **Annual Inventory Day and Month**.
+
+.. image:: count_products/annual-inventory.png
```
This image is way too wide. Replace with this new screenshot (already compressed w/ pngquant): https://drive.google.com/file/d/1pdUlVAhS7heMHzvGYEkEhQw8oZPDqTQE/view?usp=sharing

mivu-odoo reviewed (changes requested):
Finished review for `count_products.rst`

Lots of images were too wide, so I took new screenshots and linked them in my comments. I also think some images aren't necessary and should be deleted, because they are redundant or they show very simple and basic things that are easy to find.

I also fixed wording. There were some spelling errors and some sentences did not make sense. I also added transition words to help the flow of the doc and show chronological order.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
```
- Delete `Counts`
- Replace with: `counts` b/c headers should be in sentence case

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+============
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
```
- Add `in Odoo` after `adjustment`
- Replace `31th` with `31st`
- Replace `decemeber` with `December`
- Replace `But` with `However,`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
+
+The goal of cycle counts is to count more often at key locations to keep your critical stock levels
```
- Restructure sentence so it's less awkward
- Replace the sentence on Lines 9-10 `The goal of...accurate.` with `The goal of cycle counts is to keep critical stock levels accurate by counting more often at key locations.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+
+In most companies, the stock is only counted once a year. That's why by default, after making an
+inventory adjustment, the scheduled date for the next count is set on the 31th of decemeber. But
+for some businesses it's crucial to have an accurate inventory count.
```
Add `at all times` after `count`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+Configuration
+=============
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
```
- Delete `That means that`
- Capitalize first letter of `the` in `the frequency`
- Replace `on` with `by`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+=============
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
+the storage location. That also means that to use cycle counts you need to activate the setting
```
- Replace the sentence on Lines 16-17 `That also means...**Storage Locations**.` with `To activate storage locations, go to :menuselection:'Inventory --> Configuration --> Settings --> Warehouse' and activate the :guilabel:'Storage Locations' setting.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+   :align: center
+   :alt: Enable Storage Locations in Odoo settings.
+
+Then, go to the locations via :menuselection:`Inventory --> Configuration --> Locations`.
```
- Replace `Then,` with `To change a location's inventory frequency, first,`
- Replace `via` with `by clicking`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+
+In Odoo, cycle counts are location-based. That means that the frequency of the counts is defined on
+the storage location. That also means that to use cycle counts you need to activate the setting
+**Storage Locations**.
```
Add a sentence `Next, click :guilabel:'Save' to apply the setting.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+.. image:: cycle_counts/storage-locations-setting.png
+   :align: center
+   :alt: Enable Storage Locations in Odoo settings.
+
```
Add a section title here `Change the inventory frequency`, use H2 formatting

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+
+Then, go to the locations via :menuselection:`Inventory --> Configuration --> Locations`.
+
+Open a location and change the Inventory Frequency. In this example we want to
```
- Replace the sentence `Open a...Inventory Frequency.` with `Then, click on a location to open the location settings. Next, click on :guilabel:'Edit'. In the :guilabel:'Inventory Frequency (Days)' field, set the number of days.`
- Replace the sentence on Lines 25-26 `In this example...every 30 days.` with `For example, a location that needs an inventory count every 30 days would set the :guilabel:'Inventory Frequency (Days)' value to '30'.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+Then, go to the locations via :menuselection:`Inventory --> Configuration --> Locations`.
+
+Open a location and change the Inventory Frequency. In this example we want to
+count items in this loctation every 30 days. That means that once an adjustment
```
- Replace the sentence on Lines 26-27 `That means that...30 days after.` with `Once the value is entered, click :guilabel:'Save' to apply the setting to the location. Now, once an inventory adjustment is applied to this location, the next scheduled count date will be automatically set based on the number of days in the :guilabel:'Inventory Frequency (Days)' setting.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst:
```diff
@@ -0,0 +1,32 @@
+is applied on this location, the next scheduled date for this line will be 30 days
+after.
+
+.. image:: cycle_counts/inventory-frequency.png
```
This image is way too wide, replace with this new screenshot (already compressed w/ pngquant): https://drive.google.com/file/d/1WEXIFyh5BteqWSxvehHYzNdEjpwPEHwT/view?usp=sharing

mivu-odoo reviewed (changes requested):
Finished review on `cycle_counts.rst`

A short and sweet one. Rewrote some sentences for better flow and clarity. Replaced one image `inventory-frequency.png` that was way too wide.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+   :align: center
+   :alt: Look at the source panel showing where RFQ/MFG orders originated from.
 
 .. tip::
```
Delete the `tip` alert block and content on Lines 80-82 `Don't forget to...in the stock valuation.` I moved this information to the suggested change for Line 56 `Before creating a new reordering rule...`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+   be accounted for in the stock valuation.
 
-Make to Order 
+Make to Order
```
Uncapitalize `Order` so header is in sentence case and so it's consistent b/c I uncapitalized the other header `Replenishment report and reordering rules`

I'm on the fence about changing or leaving this one alone though, since you can argue that `Make to Order` is the *name* of the route.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
-The Make to Order configuration is available on your product form
-through your :menuselection:`Inventory --> Inventory control --> Products` (or any
-other module where products are available).
+As this configuration is recommended in particular cases, the Make to Order is a bit hidden in
```
Replace `As this configuration is recommended in particular cases` with `Since the Make to Order (MTO) route is only recommended for customized products, the route is hidden by default.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+As this configuration is recommended in particular cases, the Make to Order is a bit hidden in
+Odoo.
+
+To activate it:
```
Replace `it` with `the Make to Order (MTO) route in Odoo`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+
+To activate it:
+
+#. go to :menuselection:`Inventory --> Configuration --> Settings`
```
- Capitalize the first letter of `go`
- Add `--> Warehouse` after `Settings` in the `:menuselection:` path
- Add a period at the end of the sentence

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+To activate it:
+
+#. go to :menuselection:`Inventory --> Configuration --> Settings`
+#. activate setting **Multi-Step Routes** and save.
```
- Capitalize the first letter of `activate`
- Move the word `setting` after `Routes`
- Replace the bold formatting around `Multi-Step Routes` with a `:guilabel:`
- Replace `save` with `click :guilabel:'Save'`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+
+#. go to :menuselection:`Inventory --> Configuration --> Settings`
+#. activate setting **Multi-Step Routes** and save.
+#. go to :menuselection:`Inventory --> Configuration --> Routes`
```
- Capitalize the first letter of `go`
- Add a period at the end of the sentence

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+#. go to :menuselection:`Inventory --> Configuration --> Settings`
+#. activate setting **Multi-Step Routes** and save.
+#. go to :menuselection:`Inventory --> Configuration --> Routes`
+#. Filter on archived routes.
```
Replace `Filter on archived routes.` with `Click on :menuselection:'Filters --> Archived' to show archived routes.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+#. activate setting **Multi-Step Routes** and save.
+#. go to :menuselection:`Inventory --> Configuration --> Routes`
+#. Filter on archived routes.
+#. Unarchive MTO route: select route and pick action *Unarchive*.
```
Replace `Unarchive MTO route: select route and pick action *Unarchive*.` with `Select the checkbox next to :guilabel:'Replenish on Order (MTO)', and click on :menuselection:'Action --> Unarchive'.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+#. Unarchive MTO route: select route and pick action *Unarchive*.
+
+.. note::
+   Activating Multi-Step Routes also activate **Storage Locations** setting. If you don't need these
```
- Add `the` before `Multi-Step Routes`
- Add a `:guilabel:` to `Multi-Step Routes`
- Add `setting` after `Multi-Step Routes`
- Replace `activate` with `activates`
- Replace the bold formatting around `Storage Locations` with a `:guilabel:`
- Replace `you don't need these features` with `these features aren't applicable to the warehouse`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+
+.. note::
+   Activating Multi-Step Routes also activate **Storage Locations** setting. If you don't need these
+   features, don't forget to disable then after unarchiving the MTO route.
```
- Delete `don't forget to`
- Replace `then` with `these settings`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+   features, don't forget to disable then after unarchiving the MTO route.
 
-On the product form, under Inventory, click on "Make To Order".
+On the product form, in the Inventory tab, click on "Make To Order".
```
Replace the entire sentence with: `To set a product's procurement route to MTO, go to :menuselection:'Inventory --> Products --> Products', click on a product to open the product form, and click :guilabel:'Edit'. Then, click on the :guilabel:'Inventory' tab and in the :guilabel:'Routes' options, select :guilabel:'Replenish on Order (MTO)'. For products purchased directly from a vendor, make sure the :guilabel:'Buy' route is selected in addition to the MTO route and a vendor is configured in the :guilabel:'Purchase' tab. For products manufactured in-house, make sure the :guilabel:'Manufacture' route is selected in addition to the MTO route and a bill of materials is configured for the product. Finally, click :guilabel:'Save'.`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
 
-.. image:: strategies/strategies03.png
-    :align: center
+.. image:: strategies/replenish-on-order-product-setting.png
```
Replace image with this new screenshot: https://drive.google.com/file/d/1QQtHBBRpjcQ_FdfPWAgZnvs4v_W9f_0X/view?usp=sharing

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
-    :align: center
+.. image:: strategies/replenish-on-order-product-setting.png
+   :align: center
+   :alt: Enable the Replenish on Order (MTO) capability on product settings.
```
- Replace `capability` with `route`
- Add `the` before `product settings`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/strategies.rst:
```diff
@@ -2,75 +2,112 @@
+   :align: center
+   :alt: Enable the Replenish on Order (MTO) capability on product settings.
 
 Choice between the two options
```
I think this whole section from Lines 108-113 `Choice between the two options...use the Replenishment Report.` can be deleted. We already go over the differences between the two strategies in the `Terminology` section. This is redundant.

mivu-odoo reviewed (changes requested):
Finished review of `strategies.rst`

This was a hefty one with a lot of spelling errors. Please note that in all my comments (and all comments on other RST files), a single quotation mark denotes a backtick.

Example:
I'll write `:guilabel:'Save'` with single quotation marks b/c if I do it w/ the correct backticks, Github makes it look like this `:guilabel:`Save``

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/uom.rst:
```diff
@@ -2,106 +2,111 @@
 
-When doing a replenishment via the *Replenish* button on the product
-form, you have the possibility to use a different unit of measure.
+When doing a replenishment via the *Replenish* button on the product form, you have the possibility
```
- Delete the two images `replenish-button-on-product.png` (Line 88) and `replenish-product-uom-settings.png` (Line 92)
- Replace the one sentence on Lines 85-86 `When doing a replenishment...use a different unit of measure.` with `A request for quotation for a product can also be generated directly from the product form using the :guilabel:'Replenish' button. After clicking :guilabel:'Replenish', a replenish assistant box pops up. The purchase unit of measure can be manually edited here if needed. Then, click :guilabel:'Confirm' to create the RFQ.` [line break] `Next, click the :guilabel:'Units Forecasted' smart button on the product form and scroll down to :menuselection:'Forecasted Inventory --> Requests for quotation'. Click on the RFQ reference number to open the draft RFQ. The purchase UoM can also be edited here if needed.`

(^ single quotation marks = backticks in my suggested change)

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/uom.rst:
```diff
@@ -2,106 +2,111 @@
+   :align: center
+   :alt: Product replenishment UoM settings.
 
 Sell in bigger UoM
```
- Add `a` after `in`
- Replace `bigger` with `different`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/uom.rst:
```diff
@@ -2,106 +2,111 @@
-You can choose the unit of measure on the sale order document and decide
-to sell the eggs by the dozen. When doing so, the price is automatically
-computed from Units UoM to adapt to the selected *UoM*.
+You can choose the unit of measure on the sale order document and decide to sell the eggs by the
```
- Delete the two images `sales-order-uom.png` (Line 103) and `delivery-order-uom-conversion.png` (Line 110)
- Replace the three sentences on Lines 99-101 and Lines 107-108 `You can choose...in our use case, the Units.` with:

`When creating a new quotation in the Sales app, Odoo automatically uses the product's specified unit of measure. However, if needed, the :guilabel:'UoM' can be manually edited on the quotation.` 
[line break]
`After the quotation is sent to the customer and confirmed into a sales order (SO), click on the :guilabel:'Delivery' smart button at the top right corner of the SO. Odoo automatically converts the unit of measure into the product's inventory unit of measure, so the :guilabel:'Demand' column of the delivery shows the converted quantity.`
[line break]
`For example, if the product's UoM on the SO was changed to 'Box of 6', but its inventory unit of measure is 'Units', the SO shows the quantity in boxes of six, and the delivery shows the quantity in units.`  

(^ single quotation marks = backticks in my suggested change)

mivu-odoo reviewed (changes requested):
Finished reviewing `uom.rst`

I found a lot of images were not needed b/c they were redundant and showed very obvious things that should be clear from the text. Other than that, I also fixed headers to be in sentence case and removed uses of familiar second-person pronouns like you, your, etc.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-To separate a delivery into different packages you will have to set the 
-done quantity to the desired package quantity then click on "PUT IN 
-PACK", do this for each package.
+To separate a delivery into different packages you will have to set the done quantity to the
```
Replace the one sentence on Lines 42-43 `To separate a delivery...do this for each package` with:

`On a delivery order, separate the products into different packages by setting the :guilabel:'Done' quantity to the desired quantity in the first package. Then, click :guilabel:'Put in Pack' to record the first package. Repeat for each package.`

(^ single quotation marks = backticks in my suggested change)

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+In Odoo, packagings are used for indicative purposes on sales/purchase orders and transfers. The
```
Add `inventory` before `transfers`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+In Odoo, packagings are used for indicative purposes on sales/purchase orders and transfers. The
+main difference between packagings and units of measure is that packagings are defined at product
```
Add `the` before `product`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-    :align: center
+.. image:: usage/product-packaging-examples.png
+   :align: center
+   :alt: Different product packaging examples
```
Add a period at the end of the sentence

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-    :align: center
+.. image:: usage/package-field-on-po.png
+   :align: center
+   :alt: Package field on purchase order
```
Add a period at the end of the sentence

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-        Another useful use of the packaging is for product reception. By
-        scanning the barcode of the packaging, Odoo adds the number of units
-        contained in the packing on the picking.
+   Another useful use of the packaging is for product reception. By scanning the barcode of the
```
- Replace `Another useful...product reception.` with `Packaging is also useful during product reception.`
- Replace `By` with `When`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/products/usage.rst:
```diff
@@ -5,68 +5,69 @@ When Should you Use Packages, Units of Measure or Special Packaging?
-        scanning the barcode of the packaging, Odoo adds the number of units
-        contained in the packing on the picking.
+   Another useful use of the packaging is for product reception. By scanning the barcode of the
+   packaging, Odoo adds the number of units contained in the packing on the picking.
```
Add `automatically` before `adds`

mivu-odoo reviewed (changes requested):
Fixed wording, formatting, and grammar issues. Some images for units of measure aren't needed. It's overkill when we could just link to the `uom.rst` in a `See Also` section at the end.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:
```diff
@@ -2,62 +2,60 @@
-At the top of my vendor bill, I’ll see a button *create landed costs*.
-I click on this button and a landed cost is automatically created. I can
-now decide on which picking those additional costs should apply.
+At the top of my vendor bill, I'll see a button *create landed costs*. I click on this button and a
```
Replace the three sentences on lines 42-44 `At the top...costs should apply.` with:

``Once the landed cost is added to the vendor bill (either by checking the :guilabel:`Landed Cost` option on the invoice line or adding a landed cost product to the bill), click the :guilabel:`Create Landed Costs` button at the top of the bill. Odoo automatically creates a landed cost record with the set landed cost pre-filled in the :guilabel:`Additional Costs` product lines. From here, decide which picking the additional costs apply to by clicking :guilabel:`Edit` and selecting the picking reference number from the :guilabel:`Transfers` drop-down menu. Finally, click :guilabel:`Save`.``

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:
```diff
@@ -2,62 +2,60 @@
-    :align: center
+.. image:: integrating_landed_costs/warehouse-transfer-landed-costs.png
+   :align: center
+   :alt: Using a warehouse transfer to cover a landed cost in the accounting journal.
```
Replace `Using` with `Use`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:
```diff
@@ -2,62 +2,60 @@
-I can now click on *Compute* and go in the tab *Valuation
-Adjustments* to see the impact on my products costs. The last step is
-to validate the landed cost.
+I can now click on *Compute* and go in the tab *Valuation Adjustments* to see the impact on my
```
Replace the two sentences on Lines 50-51 `I can now...validate the landed cost.` with:

``After setting the picking, click :guilabel:`Compute` on the landed cost record. Then, go to the :guilabel:`Valuation Adjustments` tab to see the impact of the landed costs. Finally, click :guilabel:`Validate` to post the landed cost entry to the accounting journal.``

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:
```diff
@@ -2,62 +2,60 @@
 
-I can access the journal entry that has been created by the landed cost
-by clicking on the journal entry.
+I can access the journal entry that has been created by the landed cost by clicking on the journal
```
@thomasdeleval this sentence is unclear. Is there a `Journal Entry` smartbutton that the user can click on? Or does the user have to navigate to the accounting journal to view the landed cost journal entry. 

I could not recreate the steps in Runbot to choose the picking, compute, and validate the landed cost, I got an error message `You cannot apply landed costs on the chosen Transfers(s). Landed costs can only be applied for products with FIFO or average costing method.` (Reference: https://mivu.nimbusweb.me/s/share/7482454/tkkn7m1pnvzkrvrul2b4 )

thomasdeleval replied:
@mivu-odoo 
https://drive.google.com/file/d/1g9MXA_qCWpikTU-i_QFkAz32PlK_wUcf/view?usp=drivesdk
The message is correct, the products on which to apply the landed cost must have a product category set in FIFO or AVCO.

mivu-odoo replied:
I suggested this for better clarity: https://github.com/odoo/documentation/pull/2556#discussion_r988411459

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/reporting/integrating_landed_costs.rst:
```diff
@@ -2,62 +2,60 @@
+   :alt: Landed cost journal entry
 
 .. note::
    You are not forced to start from the vendor bill, you can also go in :menuselection:`Inventory
```
Replace the sentence on Lines 61-62 `You are not forced...create the landed cost from there.` with:

``Landed cost records can also be directly created in :menuselection:`Inventory --> Operations --> Landed Costs`, it is not necessary to create a landed cost record from the vendor bill.``

mivu-odoo reviewed (changes requested):
Finished reviewing `integrating_landed_costs.rst`, had to remove uses of second-person pronouns and rewrite sections for better flow. One question for @thomasdeleval on where to view the landed cost accounting journal entry. The sentence is unclear.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst:
```diff
@@ -0,0 +1,55 @@
+.. image:: resupply_warehouses/resupply-from-second-warehouse.png
+   :align: center
+   :alt: Supply one warehouse with another in Warehouse Configuration tab.
+
```
Add a section header here `Set route on a product` and use H2 formatting

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/resupply_warehouses.rst:
```diff
@@ -0,0 +1,55 @@
+   :align: center
+   :alt: Route setting which enables a product to resupplied from a second warehouse.
+
+In this example, a reordering rule is set with a minimum of 5 units in stock and a maximum of 10
```
Replace the four sentences on Lines 37-43 `In this example...a transit location.` with:

``When a product's reordering rule is triggered and the product has the :guilabel:`Supply Product from [Warehouse Name]` route set, Odoo automatically creates two pickings. One picking is a *delivery order* from the second warehouse, which contains all the necessary products, and the second picking is a *receipt* with the same products for the main warehouse. The product move from the second warehouse to the main warehouse is fully tracked in Odoo.``
[line break]
``On the picking/transfer records created by Odoo, the :guilabel:`Source Document` is the product's reordering rule. The location between the delivery order and the receipt is a transit location.``

mivu-odoo reviewed (changes requested):
Finished reviewing `resupply_warehouses.rst`. Had to rewrite some sections for better flow and to remove uses of first and second-person pronouns.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
-configured to make your life easier. Those are based on your choice for
-shipments. Once you made your choice, head to :menuselection:`Inventory -->
-Configuration --> Routes` to see the routes Odoo generated for you.
+Some more advanced routes, such as pick-pack-ship, are already configured to make your life easier.
+Those are based on your choice for shipments. Once you made your choice, head to
+:menuselection:`Inventory --> Configuration --> Routes` to see the routes Odoo generated for you.
```
```suggestion
Some more advanced routes, such as pick-pack-ship, are also available. The user can select the route
that best fits their business needs. Once the :guilabel:`Incoming Shipments` and :guilabel:`Outgoing
Shipments` routes are set, head to :menuselection:`Inventory --> Configuration --> Routes` to see
the specific routes that Odoo generated.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
-Opening one of those routes allows you to see on what you can apply the route. 
-Here, all the selected product categories in the
-*YourCompany* warehouse are set up to follow the 3-steps route.
+Opening one of those routes allows you to see on what you can apply the route. Here, all the
+selected product categories in the *YourCompany* warehouse are set up to follow the 3-steps route.
```
```suggestion
On the :guilabel:`Routes` page, click on a route to open the route form. In the route form, the user
can view which places the route is :guilabel:`Applicable On`. The user can also set the route to
only apply on a specific :guilabel:`Company`. This is useful for multi-company environments; for
example, a user can have a company and warehouse in Country A and a second company and warehouse in
Country B.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
-you can configure custom routes in which you can define your own
-rules, but also the source and destination location of each action. As
-an example, here are the rules for the pre-configured route.
+Easier for many businesses, this process might not fit yours. Thus, you can configure custom routes
+in which you can define your own rules, but also the source and destination location of each
+action. As an example, here are the rules for the pre-configured route.
```
```suggestion
At the bottom of the route form, the user can view the specific :guilabel:`Rules` for the route.
Each :guilabel:`Rule` has an :guilabel:`Action`, a :guilabel:`Source Location`, and a
:guilabel:`Destination Location`.
```

StraubCreative commented:
Hi @mivu-odoo 

Latest changes on [095a938](https://github.com/odoo/documentation/pull/2556/commits/095a9389b613d6c6c71a60bdc91939a096e0d525), content revisions for:
- `cycle_counts.rst`
- `count_products.rst`
- `wave-transfers.rst`

Also rebased to latest commit [e21dbd0](https://github.com/odoo/documentation/commit/e21dbd01c9f4fab2507cd791cd0cefe9cdbfa2d7) and solved merge conflict for `putaway.rst`. Luckily there was only the one :)

At this point, @samueljlieber is going to jump in and help implement the next batch of content changes. 

---

@samueljlieber, I already cleaned up the RST, and @mivu-odoo did a strong pass on most of the content. What's needed now is to implement what MIVU change requests so far, however, we still have a couple docs left to review. Also if you catch anything I missed feel free to fix as well ;)

You can [get started here at this comment](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1067145869), and then work your way down, through each file, as far as you can.

When complete, please notify us here on progress, ty! 🙏 

---

**@everyone if/once the feature branch is on your computer, friendly reminder to do `git pull --rebase` each time before you work on it to ensure you're pulling in the latest content!**

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
-(here, the internal transfer from to stock to the packing area). Now,
-everything is ready to be processed so the customer can get the ordered
-items.
+As explained in this documentation introduction, the last step in the process (here the delivery
+order) is the first to be triggered, which then triggers other rules until we reach the first step
+in the process (here, the internal transfer from to stock to the packing area). Now, everything is
+ready to be processed so the customer can get the ordered items.
```
```suggestion
As explained in the introduction of the documentation, the last step in the process (for this route,
the delivery order) is the first to be triggered, which then triggers other rules until we reach the
first step in the process (here, the internal transfer from the stock to the packing area). Now,
everything is ready to be processed so the customer can get the ordered items.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
 
-In our case, the product is delivered to the customer when all the rules
-have been triggered and the transfers done.
+In our case, the product is delivered to the customer when all the rules have been triggered and
+the transfers done.
```
```suggestion
In this example, the product is delivered to the customer when all the rules have been triggered and
the transfers are done.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/concepts/use-routes.rst:
```diff
@@ -2,374 +2,327 @@
+.. image:: use_routes/transfers-status.png
    :align: center
-   :alt: view of the transfers' status when all the process is done
+   :alt: View of the transfers' status when all the process is done.
```
```suggestion
   :alt: View of the transfers' statuses when the route is completed.
```

mivu-odoo reviewed (changes requested):
Final comments on `use_routes.rst` done.

Content review still needed on
* `removal.rst`
* `putaway.rst`
* `inter_warehouse.rst`
* `warehouse_locations.rst`

mivu-odoo commented:
@StraubCreative pushed commit [c5de933](https://github.com/odoo/documentation/commit/c5de93304ed9445a41c9b979d4ed9ae10d8d9d4f) because most of the lines on `inter_warehouse.rst` that I wanted to do content review on were grayed out and I couldn't make comments. I pushed the changes on `inter_warehouse.rst` myself.

Feel free to check over my work on `inter_warehouse.rst`, as well as my comments for `use_routes.rst` [here](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1148019103). 

Files that still need final content review:
- `putaway.rst`
- `removal.rst`
- `warehouses_locations.rst`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+Location
+--------
+
+A **Location**, is a specific space within your warehouse. It can be considered as a sublocation of
+your warehouse, as a shelf, a floor, an aisle, etc. Therefore, a location is part of one warehouse
+only and it is not possible to link one location to multiple warehouses. You can configure as many
+locations as you need under one warehouse.
```
```suggestion
A **Location** is a specific space within the warehouse. It can be a sublocation of the warehouse
(a shelf, a floor, an aisle, and so on). Therefore, a location is part of one warehouse only and it
is not possible to link one location to multiple warehouses. In Odoo, as many locations can be
configured as needed under one warehouse.
```

Left the bold formatting on `Location` instead of changing it to a `:guilabel:` because here, `Location` is being introduced as a term, not a GUI element that the reader must interact with.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+only and it is not possible to link one location to multiple warehouses. You can configure as many
+locations as you need under one warehouse.
+
+There are 3 types of locations:
```
```suggestion
There are three types of locations:
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+
+There are 3 types of locations:
+
+- The **Physical Locations** are internal locations that are part of the warehouses for which you
+  are the owner. They can be the loading and unloading area of your warehouse, a shelf or a
+  department, etc.
```
```suggestion
- The **Physical Locations** are internal locations that are part of the warehouses that the company
  owns. They can be the loading and unloading areas of the warehouse, a shelf, a department, etc.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+  are the owner. They can be the loading and unloading area of your warehouse, a shelf or a
+  department, etc.
+
+- The **Partner Locations** are spaces within a customer and/or vendor's warehouse. They work the
+  same way as Physical Locations with the only difference being that you are not the owner of the
+  warehouse.
```
```suggestion
- The **Partner Locations** are spaces within a customer and/or vendor's warehouse. They work the
  same way as physical locations, with the only difference being that they are not owned by the user's
  company.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+  same way as Physical Locations with the only difference being that you are not the owner of the
+  warehouse.
+
+- The **Virtual Locations** are places that do not exist, but in which products can be placed when
+  they are not physically in an inventory yet (or anymore). They come in handy when you want to
+  place lost products out of your stock (in the **Inventory loss**), or when you want to take into
+  account products that are on their way to your warehouse (**Procurements**).
```
```suggestion
- The **Virtual Locations** are places that do not exist, but in which products can be placed when
  they are not physically in an inventory yet (or anymore). They come in handy when recording lost
  products (**Inventory Loss**), or accounting for products that are on their way to the warehouse
  (**Procurements**).
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+  place lost products out of your stock (in the **Inventory loss**), or when you want to take into
+  account products that are on their way to your warehouse (**Procurements**).
+
+In Odoo, locations are structured hierarchically. You can structure your locations as a tree,
+dependent on a parent-child relationship. This gives you more detailed levels of analysis of your
+stock operations and the organization of your warehouses.
```
```suggestion
In Odoo, locations are structured hierarchically. Locations can be structured as a tree, dependent
on a parent-child relationship. This gives more detailed levels of analysis of the stock operations
and the organization of the warehouses.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+Configuration
+=============
+
+To activate the locations, go to :menuselection:`Configuration --> Settings` and enable **Storage
+Locations**. Then, hit *Save*.
```
```suggestion
To activate locations, go to :menuselection:`Configuration --> Settings` and enable
:guilabel:`Storage Locations`. Then, click :guilabel:`Save`.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+   :alt: Enable the storage location feature in Odoo Inventory settings.
+
+.. important::
+   If you want to manage several routes within your warehouses, then enable **Multi-Step Routes**
+   and check :doc:`../../routes/concepts/use-routes`.
```
```suggestion
   To manage several routes within the warehouses, also enable :guilabel:`Multi-Step Routes` and check
   :doc:`../../routes/concepts/use-routes`.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+   If you want to manage several routes within your warehouses, then enable **Multi-Step Routes**
+   and check :doc:`../../routes/concepts/use-routes`.
+
+Creating a new warehouse
+========================
```
```suggestion
Create a new warehouse
======================
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+========================
+
+To create a warehouse, go to :menuselection:`Configuration --> Warehouse Management --> Warehouses`
+and click on **Create**.
```
```suggestion
and click on :guilabel:`Create`.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+To create a warehouse, go to :menuselection:`Configuration --> Warehouse Management --> Warehouses`
+and click on **Create**.
+
+Fill out a **Warehouse Name** and a **Short Name**. The short name is five characters maximum.
```
```suggestion
Then, fill out a :guilabel:`Warehouse Name` and a :guilabel:`Short Name`. The short name is five
characters maximum.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+   :alt: Short name field of a warehouse on Odoo Inventory.
+
+.. important::
+   The **Short Name** appears on your transfer orders and other warehouse documents. We recommend
+   using an understandable one (e.g.: WH/[first letters of location]).
```
```suggestion
   The :guilabel:`Short Name` appears on transfer orders and other warehouse documents. Odoo recommends
   using an understandable one like "WH/[first letters of location]".
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+   The **Short Name** appears on your transfer orders and other warehouse documents. We recommend
+   using an understandable one (e.g.: WH/[first letters of location]).
+
+Now, go back to your dashboard. There, new operations related to your warehouse have been
+automatically generated.
```
```suggestion
Now, go back to the :guilabel:`Inventory` dashboard. There, new operations related to the newly
created warehouse have been automatically generated.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+   :alt: Inventory app dashboard displaying new transfer types for the recently created warehouse.
+
+.. note::
+   Adding a second warehouse will automatically activate the **Locations** setting.
```
```suggestion
   Adding a second warehouse will automatically activate the :guilabel:`Locations` setting.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+.. note::
+   Adding a second warehouse will automatically activate the **Locations** setting.
+
+Creating a new location
+=======================
```
```suggestion
Create a new location
=====================
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+=======================
+
+To create a location, go to :menuselection:`Configuration --> Warehouse Management --> Locations`
+and click on **Create**.
```
```suggestion
and click on :guilabel:`Create`.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst:
```diff
@@ -0,0 +1,91 @@
+To create a location, go to :menuselection:`Configuration --> Warehouse Management --> Locations`
+and click on **Create**.
+
+Fill out a **Location Name** and a **Parent Location** and hit *Save*.
```
```suggestion
Then, fill out a :guilabel:`Location Name` and a :guilabel:`Parent Location` and click
:guilabel:`Save`.
```

mivu-odoo reviewed (changes requested):
CC: @StraubCreative 

@samueljlieber `warehouses_locations.rst` final content review done.

Two files left that need final content review:
* `putaway.rst`
* `removal.rst`

samueljlieber commented:
Hi @StraubCreative and @mivu-odoo! This was a doozy! I can tell this was a lot of work to put together, thanks for making the edits so clear 👍

In commit 96e45c4 I did the following:

1. [Updated strategies.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1067145869)
2. [Updated uom.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1074516717)
3. [Updated usage.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1074886839)
4. [Updated integrating_landed_costs.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1082525377)
5. [Updated resupply_warehouses.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1082909577)
6. [Updated use_routes.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1148019103)
7. [Updated warehouses_locations.rst](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1148377240)

@mivu-odoo I did review `inter_warehouse.rst` per your [comment](https://github.com/odoo/documentation/pull/2556#issuecomment-1284687399), everything looks good there 🙂

Awaiting the [two files left that need final content review](https://github.com/odoo/documentation/pull/2556#pullrequestreview-1148377240):

- `putaway.rst`
- `removal.rst`

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
    :align: center
+   :alt: Activate Multi-Step Routes in Inventory configuration settings.
 
 Setting up a Putaway Rule
```
```suggestion
Setting up a putaway rule
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
 ==========================
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in different locations to maintain product quality. 
+In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
+different locations depending on several factors like frequency, size, product category, need to be
+in specific environment etc...
```
```suggestion
In some cases, like for a retail shop storing fruits and vegetables, products should be stored in
different locations depending on several factors like frequency, size, product category, specific
environment needs, and so on.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+in specific environment etc...
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+Let's suppose there are one warehouse location *WH/Stock* with following sub-locations:
```
```suggestion
In this example, suppose there is one warehouse location, **WH/Stock**, with the following
sub-locations:
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+  - WH/Stock/Shelf 2/Small Refrigerator
+- WH/Stock/Shelf 3
 
 To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
 to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
 rule indicating the main location the product will enter before being redirected to the right
 location.
```
```suggestion
Manage those locations with putaway rules. To create a putaway rule, open the :guilabel:`Inventory` app
and go to :menuselection:`Configuration --> Putaway Rules`. Then, click on :guilabel:`Create` and
configure a putaway rule that indicated the main location the product will enter before being
redirected to the right storage location.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
-.. note::
-   The putaway rules can be defined either per product or per product category.
+.. important::
+   The putaway rules can be defined either per product/product category and or Package Type
+   (Packages setting must be enabled for that). Putaway rules are read sequetially until a match is
+   found.
```
```suggestion
   The putaway rules can be defined either per product/product category and/or package type (the
   :guilabel:`Packages` setting must be enabled for that). Putaway rules are read sequentially until a
   match is found.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+
+Take the following example:
+
+- If I receive water (category All/drinks), whatever the package, it will be redirected to
+  WH/Stock/Shelf 2/Small Refrigerator
+- If I receive orange juice cans in a box, it will be redirected to WH/Stock/Shelf 2
+- If I receive water or apple juice in a box, it will be redirected to WH/Stock/Shelf 3
+- If I receive a Pallet of Lemonade cans, it will be redirected to WH/Stock/pallets/PAL1
```
```suggestion
- If water (category All/drinks) is received, whatever the package, it will be redirected to
  WH/Stock/Shelf 2/Small Refrigerator.
- If orange juice cans, packaged in boxes, are received, they will be redirected to
  WH/Stock/Shelf 2.
- If water or apple juice bottles, packaged in boxes, are received, they will be redirected to
  WH/Stock/Shelf 3.
- If a pallet of lemonade cans are receieved, it will be redirected to WH/Stock/Pallets/PAL1.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+Using Storage Categories
+========================
+
+A **Storage category** is an extra location attribute. It allows to define the quantity of products
+that can be stored in the location and how the location will be selected with putaway rules.
```
```suggestion
A *Storage Category* is an extra location attribute. Storage categories allow the user to define the
quantity of products that can be stored in the location and how the location will be selected with
putaway rules.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+Create a Storage Category
+=========================
+
+In the *Inventory* app go to :menuselection:`Configuration --> Storage Categories`. Hit *Create*.
```
```suggestion
To create a storage category, go to :menuselection:`Inventory --> Configuration --> Storage
Categories` and click :guilabel:`Create`. Then, click :guilabel:`Save` and click
:guilabel:`Storage Categories` or go to :menuselection:`Configuration --> Storage Categories` to
create a new storage category.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+==============
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but redirected to the right location automatically, thanks to putaway rules. This information is available from *Inventory Report*, under the reporting menu.
+In the *Inventory* app, go to :menuselection:`Configuration --> Settings` and activate the *Storage
+Categories*. By doing so, the *Storage Locations* will be automatically activated.
```
```suggestion
In the :guilabel:`Inventory` app, go to :menuselection:`Configuration --> Settings` and activate the
:guilabel:`Storage Categories` feature. By doing so, the :guilabel:`Storage Locations` feature is
also automatically activated.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
    :align: center
+   :alt: Create Storage Categories inside Odoo Inventory configuration settings.
+
+Fill a name. You can limit the capacity by weight, by product or package type. The *Allow New
+Product* define when the location is considered available to store a product:
```
```suggestion
First, click :guilabel:`Create` and type a name for the storage category. 

Then, there are options to limit the capacity by weight, by product, or by package type. The
:guilabel:`Allow New Product` field defines when the location is considered available to store a
product:
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+Fill a name. You can limit the capacity by weight, by product or package type. The *Allow New
+Product* define when the location is considered available to store a product:
+
+- If location is empty: a product can be added there only if the location is empty
+- If products are the same: a product can be added there only if the same product is already there
+- Allow mixed products: several different products can be stored in this location at the same time
```
```suggestion
- :guilabel:`If location is empty`: a product can be added there only if the location is empty.
- :guilabel:`If products are the same`: a product can be added there only if the same product is
  already there.
- :guilabel:`Allow mixed products`: several different products can be stored in this location at the
  same time.
```

samueljlieber commented:
[81d5f31](https://github.com/odoo/documentation/commit/81d5f31cc240a15d3f4a4141c2cc47811dd2b7da) to correct commit message.

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+- If products are the same: a product can be added there only if the same product is already there
+- Allow mixed products: several different products can be stored in this location at the same time
+
+Now we can link a storage category to a location.
```
```suggestion
Once the storage category settings are saved, the storage category can be linked to a location.
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
    :align: center
+   :alt: When a Storage Category is created, it can be linked to a warehouse location.
+
+Storage Categories in putaway rules
```
```suggestion
Storage categories in putaway rules
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+Storage Categories in putaway rules
+===================================
+
+Lets continue our example from above we applied our 'High Frequency Pallets' on locations PAL1 and
+PAL2 and reworked the putaway rules as follows.
```
```suggestion
To continue the example from above, apply the "High Frequency Pallets" on the PAL1 and PAL2
locations and rework the putaway rules as follows:
```

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:
```diff
@@ -2,44 +2,112 @@
+Lets continue our example from above we applied our 'High Frequency Pallets' on locations PAL1 and
+PAL2 and reworked the putaway rules as follows.
+
+Assume I receive 1 pallet of lemonade cans:
+
+- if PAL1 and PAL2 are empty, the pallet will be redirected to Stock/pallets/PAL1
+- if PAL1 is full, the pallet will be redirected to stock/pallets/PAL2
+- if PAL1 and 2 are full, the pallet will be redirected to stock/Pallets
```
```suggestion
Assume receive one pallet of lemonade cans is received:

- If PAL1 and PAL2 are empty, the pallet will be redirected to WH/Stock/Pallets/PAL1.
- If PAL1 is full, the pallet will be redirected to WH/Stock/Pallets/PAL2.
- If PAL1 and 2 are full, the pallet will be redirected to WH/Stock/Pallets.
```

StraubCreative replied:
removed first `receive` in line 105, kept the 2nd.

mivu-odoo reviewed (changes requested):
CC: @StraubCreative 

Hi @samueljlieber! Thank you for making those edits. I finished one more for you: `putaway.rst`. Only one more left that needs final content review, `removal.rst`!

StraubCreative commented:
@mivu-odoo `putaway.rst` updates applied on [79f76ea](https://github.com/odoo/documentation/pull/2556/commits/79f76ea16dd3045f128e46c2e859721ddad7e958).

All that's left is `removal.rst` :)

cc: @samueljlieber 

---
Reminder to `git pull --rebase` the latest change(s) 💡

mivu-odoo commented on content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst:
```diff
@@ -1,207 +1,217 @@
 
-By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot 
-numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory --> 
+By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot
+numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory -->
 Master Data --> Lots/Serial Numbers`.
```
```suggestion
By activating the :guilabel:`Expiration Dates` feature, it becomes possible to define different
expiration dates on the serial/lot numbers that will be used in :abbr:`FEFO (First Expired, First
Out)`. These expiration dates can be set by going to :menuselection:`Inventory --> Products -->
Lots/Serial Numbers`.
```

mivu-odoo reviewed (changes requested):
Final one 🥳

samueljlieber commented:
@mivu-odoo & @StraubCreative all set with the `removal.rst` updates! 🙂

StraubCreative commented:
Thanks for all of your hard work on this @mivu-odoo and @samueljlieber !
Tagging @odoo/doc-review for final review 🚀

AntoineVDV reviewed (changes requested):
Nice work :)

samueljlieber commented:
@AntoineVDV Thank you for the suggestions! They have been implemented in [f7c003b](https://github.com/odoo/documentation/commit/f7c003b0bac29e9685e52efe4bb8f9ed5eedb4b0).

AntoineVDV reviewed (approved):
@robodoo r+
